components/Layout/Layout.tsx

```diff
@@ -68,7 +68,8 @@ export default function AppLayout({ children }: AppLayoutProps) {
                 </HeaderStateContext>
                 {presentPath !== '/match' &&
                   presentPath !== '/manual' &&
-                  presentPath !== '/store' && (
+                  presentPath !== '/store' &&
+                  !presentPath.startsWith('/party') && (
                     <div className={styles.buttonContainer}>
```

Comment:
만약 그렇게 적으시면 이렇게 통일하는 게 좋지 않을까요?
```suggestion
                {!presentPath.startsWith('/match') &&
                  !presentPath.startsWith('/manual') &&
                  !presentPath.startsWith('/store') &&
                  !presentPath.startsWith('/party') && (
                    <div className={styles.buttonContainer}>
```

Reply:
나머지 부분들은 제가 작성한게 아니어서 일단은 그냥 됬습니다.
startsWith함수 사용한건 party만 페이지가 여러개여서 사용했습니다.
Layout.tsx이 부분부분수정된게 많아서 나중에 리팩토링할 때 한번에 수정하면 좋을 것 같아요